src/main/java/com/google/devtools/build/lib/runtime/ExperimentalStateTracker.java

```diff
@@ -192,6 +192,10 @@ void progressReceiverAvailable(ExecutionProgressReceiverAvailableEvent event) {
     executionProgressReceiver = event.getExecutionProgressReceiver();
   }
 
+  private String toRegularPluralizedString(int val, String msg) {
```

Comment:
This is unused?

Reply:
removed

Reply:
file changes reverted - since they are no longer relevant